### ISSUE TITLE
Bug installing ASReview windows user

### ISSUE THREAD

**Issue by MohsenAskar** [asreview/asreview#599]
I am trying to run ASReview on windows 10. The installation went successfully, but when trying to run asreview from cmd the following error appears:

C:\Users\mas082>asreview lab
c:\users\mas082\anaconda3\envs\tensor\lib\site-packages\numpy\_distributor_init.py:32: UserWarning: loaded more than 1 DLL from .libs:
c:\users\mas082\anaconda3\envs\tensor\lib\site-packages\numpy\.libs\libopenblas.NOIJJG62EMASZI6NYURL6JBKM4EVBGM7.gfortran-win_amd64.dll
c:\users\mas082\anaconda3\envs\tensor\lib\site-packages\numpy\.libs\libopenblas.PYQHXLVVQ7VESDPUVUADXEVJOBGHJPAY.gfortran-win_amd64.dll
c:\users\mas082\anaconda3\envs\tensor\lib\site-packages\numpy\.libs\libopenblas.WCDJNK7YVMPZQ2ME2ZZHJJRJ3JIKNDB7.gfortran-win_amd64.dll
  stacklevel=1)
2021-05-09 23:37:57.861474: W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found
2021-05-09 23:37:57.861685: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
Traceback (most recent call last):
  File "c:\users\mas082\anaconda3\envs\tensor\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\mas082\anaconda3\envs\tensor\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\mas082\AppData\Roaming\Python\Python36\Scripts\asreview.exe\__main__.py", line 7, in <module>
  File "C:\Users\mas082\AppData\Roaming\Python\Python36\site-packages\asreview\__main__.py", line 80, in main
    entry.load()().execute(sys.argv[2:])
  File "c:\users\mas082\anaconda3\envs\tensor\lib\site-packages\pkg_resources\__init__.py", line 2472, in load
    return self.resolve()
  File "c:\users\mas082\anaconda3\envs\tensor\lib\site-packages\pkg_resources\__init__.py", line 2478, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "C:\Users\mas082\AppData\Roaming\Python\Python36\site-packages\asreview\entry_points\__init__.py", line 17, in <module>
    from asreview.entry_points.lab import LABEntryPoint
  File "C:\Users\mas082\AppData\Roaming\Python\Python36\site-packages\asreview\entry_points\lab.py", line 19, in <module>
    from asreview.webapp.run_model import main as main_run_model
  File "C:\Users\mas082\AppData\Roaming\Python\Python36\site-packages\asreview\webapp\run_model.py", line 26, in <module>
    from asreview.webapp.sqlock import SQLiteLock
  File "C:\Users\mas082\AppData\Roaming\Python\Python36\site-packages\asreview\webapp\sqlock.py", line 17, in <module>
    import sqlite3
  File "c:\users\mas082\anaconda3\envs\tensor\lib\sqlite3\__init__.py", line 23, in <module>
    from sqlite3.dbapi2 import *
  File "c:\users\mas082\anaconda3\envs\tensor\lib\sqlite3\dbapi2.py", line 27, in <module>
    from _sqlite3 import *
ImportError: DLL load failed: The specified module could not be found.

What should I do? In plain English :D 

Thanks in advance

Version info:
Windows 10
python 3.6.12
tensorflow 2.3.0
numpy 1.17.0
ASReview version 0.17

**Comment by J535D165** [comment]
Hi @MohsenAskar, thanks for the report. 

Your issue is unrelated to the installation of ASReview I think. This is the issue you are facing: https://stackoverflow.com/questions/54876404/unable-to-import-sqlite3-using-anaconda-python.

Feel free to reopen this issue.
